cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -611,6 +611,8 @@
       ImmutableMap.of(Constants.Metrics.Tag.NAMESPACE, NamespaceId.SYSTEM.getNamespace(),
                       Constants.Metrics.Tag.COMPONENT, Constants.Service.TRANSACTION);
 
+    public static final String EMIT_PROGRAM_CONTAINER_METRICS_PREFERENCE = "system.metrics.enabled";
+    public static final String MR_METRICS_TASK_LEVEL_PREFERENCE = "system.task.level.metrics.enabled";
```

Comment:
Why need this?? Task runtime arg should be just defined by scope. E.g. to disable all task level metrics, provide `task.*.metrics.enabled = false` to the runtime argument.